engines/pyfunc-ensembler-service/app.Dockerfile

```diff
@@ -8,7 +8,8 @@ ARG FOLDER_NAME
 RUN gsutil -m cp -r ${MODEL_URL} .
 
 # Install dependencies required by the user-defined ensembler
-RUN /bin/bash -c ". activate ${CONDA_ENV_NAME} && conda env update --name ${CONDA_ENV_NAME} --file /${HOME}/${FOLDER_NAME}/conda.yaml"
+RUN /bin/bash -c ". activate ${CONDA_ENV_NAME}"
+RUN conda env update --name ${CONDA_ENV_NAME} --file ./${FOLDER_NAME}/conda.yaml
```

Comment:
Does this actually work? The reason why it was done inline is because the states do not share when running in a new docker layer. i.e. running `/bin/bash -c ". activate ${CONDA_ENV_NAME}"` then closing the shell and then running `conda env update --name ${CONDA_ENV_NAME} --file ./${FOLDER_NAME}/conda.yaml`.

If this works isn't line 11 redundant?

Reply:
Hmm it actually works, so you're right I think - line 11 is indeed probably redundant. I'll give it a try and see how it goes.

Reply:
Okay it works as usual. I've pushed a new commit without that line.